packages/instantsearch-ui-components/src/components/recommend-inner/DefaultHeader.tsx

```diff
@@ -0,0 +1,26 @@
+/** @jsx createElement */
+import { cx } from '../../lib';
+
+import type { InnerComponentProps, Renderer } from '../../types';
+
+export function createDefaultHeaderComponent({ createElement }: Renderer) {
+  return function DefaultHeader<TObject>(
+    userProps: InnerComponentProps<TObject>
+  ) {
+    const { classNames = {}, recommendations, translations } = userProps;
```

Comment:
Could this be spread directly in function parameters?

Reply:
Sure, however I followed the example in `Hits.tsx`  thinking it was the norm in this project

Reply:
Hmm there might be a reason why we did this indeed, I recall some conversation with @Haroenv on that topic but can't recall why we did this.